config/prow/cluster/build_kubernetes-external-secrets_crd.yaml

```diff
@@ -0,0 +1 @@
+./kubernetes-external-secrets_crd.yaml
```

Comment:
Shouldn't this be a yaml file?

Reply:
This is a symlink. I meant to use the same file, but this rule:
```
 component(
        "kubernetes-external-secrets",
        "deployment",
        "rbac",
        "service",
        "crd",
        cluster = BUILD_CLUSTER,
    ),
```
kept on complaining the rule `kubernetes-external-secrets-deployment` already exists, and creating this symlink seems solving the problem